.github/renovate.json

```diff
@@ -6,7 +6,10 @@
   ],
   "regexManagers": [
     {
-      "fileMatch":  [".*\\.Dockerfile$"],
+      "fileMatch":  [
+        ".*\\.Dockerfile$",
+        ".*packaging.md$"
```

Comment:
Probably won't change anything, but shouldn't we escape the `.`, as in: `"packaging\\.md$"`.  And do you need to match `doc/foo-packaging.md` also?

Reply:
Done.